test/__init__.py

```diff
@@ -149,19 +149,6 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def notOpenSSL098(test):
-    """Skips this test for Python 3.5 macOS python.org distribution"""
-
-    @functools.wraps(test)
-    def wrapper(*args, **kwargs):
-        is_stdlib_ssl = not ssl_.IS_SECURETRANSPORT and not ssl_.IS_PYOPENSSL
-        if is_stdlib_ssl and ssl.OPENSSL_VERSION == "OpenSSL 0.9.8zh 14 Jan 2016":
```

Comment:
Looks like `ssl` module is no longer required in this module

Reply:
Thanks, fixed! The fix dismissed your review though, so you'll have to approve again